api/test/integration/test_dist_analytics.py

```diff
@@ -319,17 +510,16 @@ def write_data_file(dir_path, data):
     data_file.write_text(json.dumps(data))
 
 
-def retry_getting_resource(resource_id: str):
-    data = client.get(f"/v0/land_change/dist_alerts/analytics/{resource_id}").json()[
-        "data"
-    ]
+async def retry_getting_resource(resource_id: str, client):
+    resource = await client.get(f"/v0/land_change/dist_alerts/analytics/{resource_id}")
+    data = resource.json()["data"]
     status = data["status"]
     attempts = 1
     while status == "pending" and attempts < 10:
-        data = client.get(
-            f"/v0/land_change/dist_alerts/analytics/{resource_id}"
-        ).json()["data"]
+        resp = await client.get(f"/v0/land_change/dist_alerts/analytics/{resource_id}")
+        data = resp.json()["data"]
         status = data["status"]
+        time.sleep(1)
         attempts += 1
     if attempts >= 10:
         pytest.fail("Resource stuck on 'pending' status")
```

Comment:
Oye. This test might take 10 seconds? :(

Reply:
Going to revisit the multiple AOI tests to make sure they don't use real data and can run fast.

Reply:
Just a reminder that:
1. integration tests are slow
2. we shouldn't have many of them
3. focus on unit/collaboration testing and small datasets for quick feedback as @jterry64 mentions

Reply:
Will wait on this, I think Gary came up with a good refactoring to avoid slow tests like this